lldb/test/API/python_api/type/TestTypeList.py

```diff
@@ -150,6 +150,23 @@ def test(self):
         invalid_type = task_type.FindDirectNestedType(None)
         self.assertFalse(invalid_type)
 
+        # Check that FindDirectNestedType works with types from AST
+        pointer = frame0.FindVariable("pointer")
+        pointer_type = pointer.GetType()
+        self.assertTrue(pointer_type)
+        self.DebugSBType(pointer_type)
+        pointer_info_type = pointer_type.template_args[0]
```

Comment:
```suggestion
        pointer_info_type = pointer_type.template_args[1]
```

then the tests works :)

Reply:
You're absolutely right. Thank you!